ext/filter/filter.stub.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+function filter_has_var(int $type, string $variable_name): bool {}
+
+/**
+ * @param mixed|null $options
```

Comment:
Isn't `null` part of `mixed`?

Reply:
Fixed on bf8d3e35f51a4aa098ec8e533c3e06e2539ca263